documentation/asciidoc/topics/proc_deploying_cryostat.adoc

```diff
@@ -19,29 +19,39 @@ Depending on your requirements, you can store and analyze your recordings using
 ----
 include::yaml/cryostat.yaml[]
 ----
++
+[NOTE]
+====
+The Cryostat Operator requires cert-manager for traffic encryption.
+If the cert-manager is enabled but not installed, the deployment fails.
+For details, see the {cryostat_installation} guide.
+====
 
 . Wait for the `Cryostat` CR to be ready.
 +
 [source,options="nowrap",subs=attributes+]
 ----
-{oc} wait -n <namespace> --for=condition=MainDeploymentAvailable cryostat/test-cryostat-provisioning-cs
+{oc} wait -n <namespace> --for=condition=MainDeploymentAvailable cryostat/cryostat-sample
 ----
 
 . Open the Cryostat `status.applicationUrl`.
 +
 [source,options="nowrap",subs=attributes+]
 ----
-{oc} -n <namespace> get cryostat {example_crd_name}-cs
+{oc} -n <namespace> get cryostat cryostat-sample
 ----
 
-. Retrieve the Operator user credentials to authenticate client JMX connections in the Cryostat UI.
+. In the Cryostat UI, retrieve the Operator user credentials to authenticate client JMX connections.
```

Comment:
I'd keep the original one, we are not retrieving Operator credentials from Cryostat UI

Reply:
Yes you're right. Now it's updated :)